/home/wang9/anaconda3/envs/torch_1.11/lib/python3.10/site-packages/torch/utils/data/dataloader.py:563: UserWarning: This DataLoader will create 16 worker processes in total. Our suggested max number of worker in current system is 8, which is smaller than what this DataLoader is going to create. Please be aware that excessive worker creation might get DataLoader running slow or even freeze, lower the worker number to avoid potential slowness/freeze if necessary.
  warnings.warn(_create_warning_msg(
0it [00:00, ?it/s]1it [00:47, 47.73s/it]2it [00:48, 20.11s/it]3it [00:49, 11.23s/it]4it [00:49,  7.07s/it]5it [00:50,  4.76s/it]6it [00:51,  3.38s/it]7it [00:51,  2.50s/it]8it [00:52,  1.92s/it]9it [00:53,  1.53s/it]10it [00:53,  1.27s/it]11it [00:54,  1.09s/it]12it [00:55,  1.04it/s]13it [00:55,  1.14it/s]14it [00:56,  1.22it/s]15it [00:57,  1.28it/s]16it [00:58,  1.33it/s]17it [01:22,  7.77s/it]18it [01:26,  6.76s/it]19it [01:27,  4.94s/it]20it [01:27,  3.67s/it]21it [01:28,  2.77s/it]22it [01:29,  2.14s/it]23it [01:30,  1.71s/it]24it [01:30,  1.40s/it]25it [01:31,  1.18s/it]26it [01:32,  1.03s/it]27it [01:32,  1.08it/s]28it [01:33,  1.18it/s]29it [01:34,  1.26it/s]30it [01:34,  1.31it/s]31it [01:35,  1.36it/s]32it [01:36,  1.39it/s]33it [02:02,  8.57s/it]34it [02:08,  7.62s/it]35it [02:09,  5.53s/it]36it [02:09,  4.08s/it]37it [02:10,  3.06s/it]38it [02:11,  2.34s/it]39it [02:11,  1.84s/it]40it [02:12,  1.50s/it]41it [02:13,  1.25s/it]42it [02:13,  1.08s/it]43it [02:14,  1.05it/s]44it [02:15,  1.15it/s]45it [02:15,  1.23it/s]46it [02:16,  1.29it/s]47it [02:17,  1.34it/s]48it [02:17,  1.38it/s]49it [02:37,  6.35s/it]50it [02:41,  5.68s/it]51it [02:42,  4.18s/it]52it [02:42,  3.13s/it]53it [02:43,  2.39s/it]54it [02:44,  1.88s/it]55it [02:44,  1.51s/it]56it [02:45,  1.26s/it]57it [02:46,  1.08s/it]58it [02:46,  1.04it/s]59it [02:47,  1.14it/s]60it [02:48,  1.23it/s]61it [02:48,  1.30it/s]61it [02:48,  2.77s/it]
Traceback (most recent call last):
  File "/usr/local/bin/jobinfo.py", line 14, in <module>
    gpu_stats = getstats(sys.argv[1])
  File "/usr/local/bin/jobinfo.py", line 10, in getstats
    with open(fname) as stats_file:
IOError: [Errno 2] No such file or directory: '/run/gpustats.json'
